packages/js/components/src/calendar/date-picker.js

```diff
@@ -86,9 +86,12 @@ class DatePicker extends Component {
 			popoverProps = { inline: true },
 		} = this.props;
 
+		if ( ! popoverProps.position ) {
+			popoverProps.position = 'bottom center';
+		}
```

Comment:
Similarely to the previous comment:

```suggestion
		if ( ! popoverProps.position ) {
			popoverProps.placement = 'bottom';
		}
```